pkg/apis/v2v/v1alpha1/vmimportconfig_types.go

```diff
@@ -25,6 +25,33 @@ import (
 // VMImportConfigSpec defines the desired state of VMImportConfig
 type VMImportConfigSpec struct {
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// Rules on which nodes controller pod(s) will be scheduled
+	Infra NodePlacement `json:"infra,omitempty"`
```

Comment:
should this field be mandatory ?

Reply:
Based on HCO design my understanding is that if it is not there we rely on scheduler to place the controller pod on any node. In this context it is OK to have it optional.

Reply:
Added  // +optional  for both